examples/analysis/analyzer-results.txt

```diff
@@ -1,4 +1,4 @@
-Analyzing analysis_options.yaml, lib...
+Analyzing analysis...
```

Comment:
Really? "Analyzing analysis"? 😆 

Not a big deal, but maybe we could say something like `Analyzing examples/analysis...`

Reply:
Ha, this will always be the name of the directory, since `dart analyze` doesn't take an additional argument.